src/features/core/cookieConsent/ketch/browser.js

```diff
@@ -0,0 +1,83 @@
+import logger from '../../../../utils/logUtil';
+
+/* eslint-disable class-methods-use-this */
+class Ketch {
+  constructor() {
+    this.userConsentedPurposes = [];
+    this.userDeniedPurposes = [];
+
+    // updateKetchConsent callback function to update current consent purpose state
+    // this will be called from ketch ruddersatck plugin
```

Comment:
typo in word ruddersatck

Reply:
The expectation is that the consumer will load a plugin in ketch sdk too?

Reply:
yes, on the ketch side, we would register this callback function in a plugin, so each time on consent change, it would keep the consent updated without requiring the page to load again.

Reply:
Can you provide us the plugin and the steps to integrate it?

Reply:
And how do we load rudderstack after ketch is loaded and consent is given?